src/pages/MentorProfile/MentorProfile.component.tsx

```diff
@@ -168,7 +169,15 @@ const MentorProfile: React.FC = () => {
       </div>
       <div className="pb-4">
         <h2 className="text-lg font-medium ">Available mentee slots</h2>
-        <p className="font-light">{mentor?.application.noOfMentees}</p>
+        <p className="font-light">
+          {(mentor?.application.noOfMentees &&
+            mentor.mentees &&
+            mentor?.application.noOfMentees -
+              mentor?.mentees?.filter(
+                (mentee) => mentee.state === ApplicationStatus.APPROVED
+              ).length) ??
```

Comment:
Sometimes mentors might accept more students later than they mentioned in the application. Then it will show negative values. In that scenario, minimum value we should be showing is 0.

Reply:
Thanks! Assumed we don't allow that. Added the fix.